bbcdll/bbc_tcp_server.py

```
@@ -101,7 +101,15 @@ class MockCT:
             Blue = "blue"
             Colorful = "colorful"
-            BATTLE_TYPE = ['连续出击(或强化本)', '自动编队爬塔(应用操作序列设置)']
+            BATTLE_TYPE = [
+                '连续出击(或强化本)',
+                '自动编队爬塔(应用操作序列设置)',
+                '自动编队爬塔(程序自主技能宝具)',
+                '幕间物语(部分需手动)',
+                '清自由本(不包括2.7)',
+                '类主线(部分情况手动)',
+                '主线物语·大奥(部分情况手动)'
+            ]
```

Ruff (0.15.11)

[warning] Mutable default value for class attribute

(RUF012)